content/pages/privacy-policy.md

```diff
@@ -0,0 +1,105 @@
+title: Mixxx Privacy Policy
+slug: privacy-policy
+
+## Introduction
+
+Mixxx is a non-profit project, managed by volunteers.
+
+There is no business-model around Mixxx, and the only revenue we receive is through donations.
+
+For privacy policies and conformance to GDPR, a few distinctions will be done between:
+
+- Mixxx application
+- Main Mixxx website
+- Community Mixxx websites
+
+For each of the previous software/websites, we'll detail the different privacy policies.
+
+## Mixxx application
+
+The Mixxx application doesn't use any user account, and does not collect any user data, when running.
+
+Mixxx does not collect any data, nor any telemetry, when Mixxx is being run.
+However, some limited personal information could be collected or shared in 3 precise cases:
+
+- during the crash report process,
+- during updates checking,
+- for metadata retrieval.
+
+## Mixxx crash reporting
+
+The Mixxx application itself neither collects crash reports nor shares them.
+
+Crash reports are handled by the Stores/Vendors of each platforms.
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx Core Developers have access to these reports, which usually contain:
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+
+## Mixxx update mechanism
+
+The Mixxx application itself doesn't provide an update mechanism.
+
+Updates are handled by the Stores of each platforms, where the user has downloaded Mixxx from.
+Please refer to those platforms for information about the privacy policies.

```

Comment:
```suggestion
In case updates are handled by the platform stores, where the user has downloaded Mixxx from,
please refer to those platforms for information about the privacy policies.
```